src/model/block/ISelectOneResponseBlockConfig.ts

```diff
@@ -20,9 +20,35 @@
 import {IBlockConfig} from '../..'
 
 export interface ISelectOneResponseBlockConfig extends IBlockConfig {
-  prompt: string
-  question_prompt?: string
-  choices: StringMapType
+  prompt?: Uuid
+  question_prompt?: Uuid
+  choices: Record<string, Choice>
+  ivr?: IvrConfig
 }
 
-type StringMapType = {[k: string]: string}
+interface Choice {
+  name: string
+  ivr_test?: IvrTest
+  text_tests?: TextTest[]
+  prompt: Uuid
```

Comment:
Let's confirm this with @seifertk , the spec says OPTIONAL https://floip.gitbook.io/flow-specification/v/drafts-1.0.0-rc4/layers/2-mobile-primitives#choices-configuration, but old implementation is required.

- [ ] If we change this to optional, after we merge my PR, we need to make sure we don't have TS errors again when running `yarn build`